### PR TITLE
MLDB-1949 tensorflow 0.10

### PR DESCRIPTION
**Pull request by jeremybarnes**
This updates Tensorflow to version 0.10.  That version works on the Jetson TX1, making that very close to a supported platform for MLDB.

Note that now Cuda version 8.0 is now required to build MLDB.

### CODE REVIEW

File: ext/protobuf.mk [hunk 1]
```diff
@@ -24,7 +24,7 @@ $(4)/protoc: $(if $(call sne,$(1),$(HOSTARCH)),$(HOSTBIN)/protoc)
 	@echo "   $(COLOR_BLUE)[COPY EXTERN]$(COLOR_RESET)                      	protobuf3 $(1)"
 	@mkdir -p $(BUILD)/$(1)/tmp/protobuf-build
 	@cp -rf mldb/ext/protobuf/* $(BUILD)/$(1)/tmp/protobuf-build
-	@ln -sf $(PWD)/mldb/ext/gmock $(BUILD)/$(1)/tmp/protobuf-build/
+	@cp -rf $(PWD)/mldb/ext/gmock $(BUILD)/$(1)/tmp/protobuf-build/
```
Comment (FinchPowers) [inline ext/protobuf.mk]: @jraby PTI ;o)

**Comment by FinchPowers** [comment]
As-tu testé avec `make docker_mldb`?
+1

**Comment by jeremybarnes** [comment]
I will test with docker before merging

**Comment by jeremybarnes** [comment]
make docker_mldb now works

**Pull request merged**